package.json

```diff
@@ -15,7 +15,7 @@
   "author": "Ted Piotrowski",
   "license": "MIT",
   "peerDependencies": {
-    "react-leaflet": "^3.0.0"
+    "react-leaflet": "^3.0.0 | ^4.0.0"
```

Comment:
@Ciloe All the documentation I see uses the double pipe `||` for unions of version ranges. Have you tested this and can you point me to some documentation where the single `|` is explained. 

Otherwise, happy to merge this. Thank you.

Reply:
Yes You are right ! My bad, this is the php composer syntax. I will change this

Reply:
```suggestion
    "react-leaflet": "^3.0.0 || ^4.0.0"
```

Reply:
This website show the syntax at the bottom : https://semver.npmjs.com/